clr-k8s-examples/setup_system.sh

```diff
@@ -84,4 +84,3 @@ set -o nounset
 # We have potentially modified their env files, we need to restart the services.
 sudo systemctl daemon-reload
 sudo systemctl restart $RUNNER || true
```

Comment:
Should we remove `|| true` here just to make sure cri comes up? @ganeshmaharaj

Reply:
Yes. if we are not forcing it to happen above then, we should do it here.  @jwang11